practice-one/src/components/food-card.html

```diff
@@ -0,0 +1,22 @@
+<div class="food-card-container">
+  <div class="food-card-wrapper">
+    <img src="{{image}}" alt="{{imageAlt}}" />
+  </div>
+  <div class="food-price">
+    <p>{{price}}</p>
+  </div>
+  <div class="card-detail">
+    <div class="food-rating">
+      <div class="rating-1"><img src="{{rating1}}" alt="Rating 1"/></div>
+      <div class="rating-2"><img src="{{rating2}}" alt="Rating 2"/></div>
+      <div class="rating-3"><img src="{{rating3}}" alt="Rating 3"/></div>
+      <div class="rating"><img src="{{ratingStar}}" alt="Rating Star"/></div>
```

Comment:
any case `ratingStar` is different and we must pass another image here

Reply:
i understood, thanks